fiftyone/utils/sam3.py

```
@@ -250,58 +250,99 @@ def build_sam_datapoint_transform():
 
     def __init__(self, classes=None, mask_thresh=0.5):
         self.classes = None
         self.mask_thresh = mask_thresh
 
-    def __call__(
-        self,
-        output,
-        frame_size,
-    ):
-        """Returns processed model output in FiftyOne format.
+    def __call__(self, output, frame_size, query_texts=None):
```

Ruff (0.15.18)

[warning] Missing return type annotation for special method `__call__`

(ANN204)